protocol/xmrmaker/swap_state.go

```diff
@@ -452,6 +450,11 @@ func (s *swapState) SendKeysMessage() common.Message {
 	}
 }
 
+func (s *swapState) UpdateStatus(status types.Status) {
```

Comment:
doesn't need to be exported I don't think

Reply:
Fixed!